src/app_with_sec_tester/spec/setup/sec_tester.cr

```diff
@@ -0,0 +1,8 @@
+require "lucky_sec_tester"
+
+# Signup for a `NEXTPLOT_TOKEN` at
+# [NeuraLegion](https://app.neuralegion.com/signup)
```

Comment:
The company name is now called [Bright](https://brightsec.com/), but as of this comment, the Github org and this URL are still pointed to NeuraLegion... I don't want to confuse people, so this will need to change once the company's name is fully moved over.